README.md

```diff
@@ -1,2 +1,55 @@
 # go-tilepacks
-A Go-based tile downloader that saves to deduplicated mbtiles files.
+
+A Go-based tile downloader that saves to deduplicated files.
+
+## Tools
+
+### build
+
+```
+> ./bin/build -h
+Usage of ./bin/build:
+  -bounds string
+    	Comma-separated bounding box in south,west,north,east format. Defaults to the whole world. (default "-90.0,-180.0,90.0,180.0")
+  -cpuprofile string
+    	Enables CPU profiling. Saves the dump to the given path.
+  -dsn string
+    	Path, or DSN string, to output files.
+  -gzip
+    	Request gzip encoding from server and store gzipped contents in mbtiles. Will gzip locally if server doesn't do it.
+  -inverted-y
+    	Invert the Y-value of tiles to match the TMS (as opposed to ZXY) tile format.
+  -mode string
```

Comment:
In anticipation of the changes in #9 to support different "input" or "fetcher"s, can we change this to a less-ambiguous "output-mode" or something?

Reply:
Of course, I will do that shortly.

Reply:
`-mode` is now `-output-mode`